dev/ci/ci-color.sh

```diff
@@ -5,6 +5,17 @@ source ${ci_dir}/ci-common.sh
 
 Color_CI_DIR=${CI_BUILD_DIR}/color
 
+# Setup Bignums
+
+source ${ci_dir}/ci-bignums.sh
```

Comment:
I see your point here but I don't think we want to use source to install packages, please, define an install function similarly to what we do for math-comp.